docs/api_familie_klantinteracties/klantinteracties_CRUD/README.md

```diff
@@ -0,0 +1,76 @@
+# API-specificaties van de CRUD variant
+
+## Status: in ontwikkeling
+
+Deze variant wordt verder uitgewerkt in samenwerking met enkele gemeenten. 
+Uitgangspunten in deze variant is dat er een CRUD-API wordt uitgewerkt waar geen "convenience"- faciliteiten in opgenomen zijn.
+Op deze manier krijgen we de technische basis-functies in beeld en kunnen we eventueel in vervolgstappen kiezen op welke wijze we "convenience"-voorzieningengaan toevoegen om performance of developer-experience te optimaliseren. 
+
+## Changelog
+
+***Versie 0.0.2-oas3.1***
+
+  - Bij de componenten `medewerker`, `organisatorischeEenheid` en `geautomatiseerdeActor` is een groeperende property toegevoegd (actortype).
+    - Ratio: Door deze toevoeging is op eenvoudige wijze onderscheid te maken tussen de properties van het supertype (`Actor`) en de properties die bij de subtypes horen van de polymorfe resource `Actor`
+  - Bij de componenten `persoon`, `contactpersoon` en `organisatie` is een groeperende property togevoegd (partijtype).
+    - Ratio: Door deze toevoeging is op eenvoudige wijze onderscheid te maken tussen de properties van het supertype (`Partij`) en de properties die bij de subtypes horen van de polymorfe resource `Partij`
+  - de property "objectidentificator" van de component `bijlage` is hernoemd naar "bijlageidentificator" (de bijbehorende query-parameter ook)
+  - de property "objectidentificator" van de component `onderwerpobject` is hernoemd naar "onderwerpobjectidentificator" (de bijbehorende query-parameter ook)
+  - de property "actoridentifcator" van de component `actor` is hernoemd naar "actoridentificator" (herstel tyepfout) (de bijbehorende query-parameter ook)
+  - De omschrijving bij het datatype Boolean aangepast zodat duidelijk is dat waarmee true en false corresponderen.
+  - Tikfout verbeterd in de operationid van de get ../betrokkenenbijklantcontact (getbetrokkenenbijklantcontact)
+  - De datatypen van het type "string" zonder format zijn voorzien van een minLengt: 1
+    - Dit betekent dat als een property met dit datatype in de requestbody of de responsebody is opgenomen, dan moet deze ook daadwerkelijk een waarde met een minimale lengte van 1 hebben.
+    - Dit betekent ook dat als een property "required" is dat er dan verplicht ook een waarde moet worden toegekend aan die property. (Voor alle properties van een ander type dan "string" wordt dit afgedwongen door de type-declaratie.)
+  - Verwijzingen naar de code van een referentielijst zijn nu opgenomen als properties in plaats van een geneste code in de property die conform de referentielijst is genoemd.
+    - Referentielijstcomponenten zijn verwijderd.
+    - Naamgevingsconventie :  codeNaamReferentielijst
```

Comment:
Waarom niet andersom? `landcode` leest toch lekkerder dan `codeLand`. Wij hadden simpelweg `land` overigens. De spec/description maakt het formaat wel duidelijk.

Niet hele sterke gevoelens over overigens. Ik vind de toevoeging van `code` (voor of achter) ook wel beter maar ZGW had dat niet.

Reply:
Hoewel je een subtiel verschillende betekenis tussen de twee zou kunnen ervaren (`landcode` klinkt meer als een opzichzelfstaand 'ding' dan `codeLand` wat duidelijk de code van een land aanduidt, vind ik dat als we `soortActor`  vanwege leesbaarheid verkiezen boven `actorType`, we hier wat mij betreft ook de meer leesbare optie `landcode` moeten kiezen.

Reply:
heh, ik zag `soortActor` (wat wil je van de Actor? Het soort!) juist als de reden dat er gekozen was voor `codeLand` (wat wil je van Land? De code!). Wat dat betreft waren de attribuutnamen wel in lijn.

Reply:
Beide uitleggen kunnen bij nader inzien inderdaad voor consistent doorgaan ;). In dat (dus dit!) geval zou ik er geen halszaak van maken en het laten zoals het is.